python3-modules.json

```diff
@@ -0,0 +1,153 @@
+{
+    "name": "python3-modules",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [
+        {
+            "name": "python3-attrs",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"attrs\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/f2/bc/d817287d1aa01878af07c19505fafd1165cd6a119e9d0821ca1d1c20312d/attrs-22.1.0-py2.py3-none-any.whl",
+                    "sha256": "86efa402f67bf2df34f51a335487cf46b1ec130d02b8d39fd248abfd30da551c",
+                    "x-checker-data": {
+                        "type": "pypi",
+                        "name": "attrs",
+                        "packagetype": "bdist_wheel"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "python3-configobj",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"configobj\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz",
+                    "sha256": "a2f5650770e1c87fb335af19a9b7eb73fc05ccf22144eb68db7d00cd2bcb0902",
+                    "x-checker-data": {
+                        "type": "pypi",
+                        "name": "configobj"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "python3-pillow",
```

Comment:
you don't need this one, pillow is already installed by ttkthemes below.

Reply:
I felt it was more maintainable to leave that file as completely unmodified output of `python3 flatpak-pip-generator --checker-data <LIST OF DIRECT DEPENDENCIES>` so it can be regenerated purely by re-running the command.

As such, removing that one would *increase* the maintainership burden by complicating the instructions to the maintainers.